docs/develop/api-reference/sorted-set-collections.md

```diff
@@ -271,3 +271,45 @@ Example:
 | Score           | Signed double 64-bit float   | Score the element. |
 
 A SortedSetElement can exist by itself or as part of an array of SortedSetElements
+
+### SortedSetLength
+Get the length of an existing sorted set item
+
+| Name           | Type         | Description                                |
+|----------------| ------------ |--------------------------------------------|
+| cacheName      | String       | Name of the cache.                         |
+| sortedSetName | String       | Name of the sorted set item to be checked. |
+
+<details>
+  <summary>Method response object</summary>
+
+* Hit
+    * `length()`: Number
+* Miss
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SortedSetLengthByScore
+For an existing sorted set item, finds all of the values between the specified min and max score and 
+
+| Name           | Type         | Description                                |
+|----------------| ------------ |--------------------------------------------|
+| cacheName      | String       | Name of the cache.                         |
+| sortedSetName | String       | Name of the sorted set item to be checked. |
+| minScore | Optional[double]   | The inclusive low score of the results. Default is `-inf`, ie include through the lowest score. |
```

Comment:
Are we finalizing this?

Reply:
@allenheltondev are you cool with this for now?  I think this is what it needs to look like here, in order to match existing SortedSet functions.  I can sync w/you on the final shape of the JS SDK API when we add it there.

Reply:
Yeah this is fine, but it leaves me with an existential crisis about these docs in general. If it's different for all our SDKs and end users don't call these APIs directly - then what is this documenting? The back-end that users don't see?

Reply:
@allenheltondev I will be adding code snippets for all the languages for all of the APIs very soon.  The purpose of this PR for me is to get these in place so i have a place to add the snippet react tags, so I can then have my team start filling in the code samples.